website/content/docs/k8s/installation/install.mdx

```diff
@@ -104,6 +104,17 @@ NAME: consul
 ...
 ```
 
+### Install Consul on OpenShift clusters
+
+Red Hat OpenShift is a security-conscious, opinionated wrapper for Kubernetes. To install Consul on OpenShift-managed Kubernetes, add the following parameters to the `global` field of your values file:
+
+```yaml
+  openshift:
+    enabled: true
+ ```
```

Comment:
I think it's clearer for people who are scanning to include the `global` as the root. They may not see the indentation immediately and it can mess them up if they are copy/pasting. 

```suggestion
[Red Hat OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift) is a security-conscious, opinionated wrapper for Kubernetes. To install Consul on OpenShift-managed Kubernetes, set `global.openshift.enabled=true` in your [custom installation](#custom-installation) values file:

```yaml
global:
  openshift:
    enabled: true
```
```